src/posts/components/PostsFilters.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { usePostsStore } from "../posts-store";
+
+const PostsFilters: React.FC = () => {
+  const { setStatusFilter, setSearchFilter, statusFilter, searchFilter } =
+    usePostsStore();
+
+  return (
+    <div className="mb-3 d-flex gap-3">
+      {/*filter by status*/}
+      <select
```

Comment:
since we are using UI package, please use its component to get consistent design, so use PrimeReact select component instead